index.css

```diff
@@ -0,0 +1,154 @@
+@media (min-width: 400px) {
+  .header_container {
+    position: absolute;
+    width: 375px;
+    height: 92px;
+
+    /* left: 0px;
+    top: 0px; */
```

Comment:
Please don't leave commented out code on your files. Kindly remove these two lines to keep your code clean.